Riot/Routers/NavigationRouter.swift

```diff
@@ -221,6 +221,36 @@ final class NavigationRouter: NSObject, NavigationRouterType {
         self.didPushViewController(controller)
     }
     
+    func push(_ modules: [PresentableModule], animated: Bool) {
+        MXLog.debug("[NavigationRouter] Push modules \(modules)")
```

Comment:
How come the implementation of this method doesn't simply map to a `Presentable` + completion and then call the method above?

Reply:
Ah it would lead to bounces. We just need to see the latest module when pushing, not every module's push one by one.

Reply:
Makes sense :)